framework/src/media/audio/audio_manager.h

```diff
@@ -376,6 +436,149 @@ audio_manager_result_t set_input_audio_gain(uint8_t gain);
+audio_manager_result_t get_stream_in_policy(audio_manager_stream_policy_t *policy);
+
+/****************************************************************************
+ * Name: get_stream_out_policy
+ *
+ * Description:
+ *   Get policy of actual stream out
+ *
+ * Input parameter:
+ *   policy : current policy to be returned
+ *
+ * Return Value:
+ *   On success, AUDIO_MANAGER_SUCCESS. Otherwise, a negative value.
+ ****************************************************************************/
+audio_manager_result_t get_stream_out_policy(audio_manager_stream_policy_t *policy);
+
+/****************************************************************************
+ * Name: change_stream_in_device
+ *
+ * Description:
+ *   Change actual stream in device with given values
+ *
+ * Input parameter:
+ *   card_id : id of sound card, device_id : id of device
+ *
+ * Return Value:
+ *   On success, AUDIO_MANAGER_SUCCESS. Otherwise, a negative value.
+ ****************************************************************************/
+audio_manager_result_t change_stream_in_device(int card_id, int device_id);
+
+/****************************************************************************
+ * Name: change_stream_out_device
+ *
+ * Description:
+ *   Change actual stream out device with given values
+ *
+ * Input parameter:
+ *   card_id : id of sound card, device_id : id of device
+ *
+ * Return Value:
+ *   On success, AUDIO_MANAGER_SUCCESS. Otherwise, a negative value.
+ ****************************************************************************/
+audio_manager_result_t change_stream_out_device(int card_id, int device_id);
+
+/****************************************************************************
+ * Name: get_stream_in_id
+ *
+ * Description:
+ *   Get card & device id of actual stream in device
+ *
+ * Input parameter:
+ *   card_id : id of sound card to be revealed , device_id : id of device to be revealed
+ *
+ * Return Value:
+ *   On success, AUDIO_MANAGER_SUCCESS. Otherwise, a negative value.
+ ****************************************************************************/
+audio_manager_result_t get_stream_in_id(int *card_id, int *device_id);
+
+/****************************************************************************
+ * Name: get_stream_out_id
+ *
+ * Description:
+ *   Get card & device id of actual stream out device
+ *
+ * Input parameter:
+ *   card_id : id of sound card to be revealed , device_id : id of device to be revealed
+ *
+ * Return Value:
+ *   On success, AUDIO_MANAGER_SUCCESS. Otherwise, a negative value.
+ ****************************************************************************/
+audio_manager_result_t get_stream_out_id(int *card_id, int *device_id);
+
+/****************************************************************************
+ * Name: dump_audio_card_info
+ *
+ * Description:
+ *   Print out information of all stream out device . It will be executed properly only media debug enabled
```

Comment:
Print out information of all stream out devices . It will be properly  executed only when media debug is enabled.

Reply:
done